automated install/basic-install.sh

```diff
@@ -913,6 +905,15 @@ install_dependent_packages INSTALLER_DEPS[@]
 # Install packages used by the Pi-hole
 install_dependent_packages PIHOLE_DEPS[@]
```

Comment:
Yeah, that's a fair comment. 

To be honest I think the pull is moot, but I submitted anyway just in case. As I said in #787, to even display the update dialogs in the first place, an existing install  needs to be detected. If an existing install is detected, then `whiptail` is already installed!